test/libsolidity/SemanticTest.cpp

```diff
@@ -467,3 +469,16 @@ std::optional<bytes> SemanticTest::builtinSmokeTest(FunctionCall const& call)
 	}
 	return result;
 }
+
+std::optional<bytes> SemanticTest::builtinAccountBalance(FunctionCall const& call)
+{
+	soltestAssert(call.arguments.parameters.size() == 1, "");
```

Comment:
Sorry for the bit of back and forth, but the dependency on assertions tells me that perhaps the builtin list should have a `max_params` setting, so that it could show a nicer error, i.e. "too many arguments". But if error handling on the builtin side is done via asserts anyway, then disregard my comment.

Reply:
And if the only error handling is these asserts, then they should have a message, i.e. "Account address expected".

Reply:
Yes, initially a `Builtin` was a struct that contained the `std::function`. That could have been easily extended to also contain the actual parameter size. However, I think it's ok to just handle the errors in the asserts. It definitely makes sense to add the error message here.